resources/src/slack/chat_post_message.rs

```diff
@@ -16,7 +17,11 @@ pub async fn post_message(
         channel.into(),
         SlackMessageContent::new().with_text(text.into()),
```

Comment:
What happens if you append `with_thread_ts` here and it's `None`? It doesn't handle that gracefully? It's already expecting an Optional AFAIK. Would definitely be preferred over the match statement below if that works.